libs/ledger-live-common/src/families/cosmos/js-signOperation.ts

```diff
@@ -128,6 +154,7 @@ const signOperation: SignOperationFnSignature<Transaction> = ({ account, deviceI
             signedOperation: {
               operation,
               signature: signed,
+              expirationDate: undefined,
```

Comment:
This is isn't required anymore since recently, you can remove

Reply:
removed